pwndbg/gdblib/disasm/arm.py

```diff
@@ -291,4 +291,6 @@ def _parse_register(
         return target
 
 
-assistant = DisassemblyAssistant("arm")
+# Register the assistant for both ARM A-profile and ARM M-profile
+assistant = DisassemblyAssistant("arm", "cpsr")
+assistant = DisassemblyAssistant("armcm", "xpsr")
```

Comment:
Is this fine that we overwrite the variable here?

Reply:
Oh yeah - that variable is unused